components/python/pybind11/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright 2022 Daniel Chan
+#
+
+BUILD_BITS=			64
+BUILD_STYLE=		setup.py
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		pybind11
+COMPONENT_VERSION=	2.9.2
+COMPONENT_SUMMARY=	Seamless operability between C++11 and Python
+COMPONENT_PROJECT_URL=	https://pypi.org/project/$(COMPONENT_NAME)/
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:e5541f8bccf9111d1a94f7897593b55c4cf1a28d5e8cfc8225a855651f011071
+COMPONENT_ARCHIVE_URL=	$(call pypi_url)
+COMPONENT_FMRI=	library/python/pybind11
+COMPONENT_CLASSIFICATION=	Development/Python
+COMPONENT_LICENSE=      BSD
+COMPONENT_LICENSE_FILE= LICENSE
+
+PYTHON_VERSIONS=	3.9
+
```

Comment:
This package doesn't seem to provide tests. Thus,
TEST_TARGET= $(NO_TESTS)
seems to be appropriate here.